<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="TransactionInDoubtException.xml" source-language="en-US" target-language="ko-KR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5823fb45b4789450593a93ff59dbbf4da2ac285f9.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">823fb45b4789450593a93ff59dbbf4da2ac285f9</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Transactions.TransactionInDoubtException">
          <source>The exception that is thrown when an operation is attempted on a transaction that is in doubt, or an attempt is made to commit the transaction and the transaction becomes InDoubt.</source>
          <target state="translated">의심스러운 트랜잭션에서 작업이 시도되거나, 트랜잭션 커밋을 시도하고 트랜잭션이 InDoubt가 되면 throw되는 예외입니다.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Transactions.TransactionInDoubtException">
          <source>This exception is thrown when an action is attempted on a transaction that is in doubt.</source>
          <target state="translated">의심 스러운 트랜잭션에 대해 작업을 시도할 때이 예외가 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Transactions.TransactionInDoubtException">
          <source>A transaction is in doubt when the state of the transaction cannot be determined.</source>
          <target state="translated">트랜잭션이 트랜잭션 상태를 알 수 없는 경우 의심 스러운에서입니다.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Transactions.TransactionInDoubtException">
          <source>Specifically, the final outcome of the transaction, whether it commits or aborts, is never known for this transaction.</source>
          <target state="translated">특히, 트랜잭션의 최종 결과 커밋되거나 중단 하는지 여부를 알 수 없습니다이 트랜잭션에 대 한 합니다.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Transactions.TransactionInDoubtException">
          <source>This exception is also thrown when an attempt is made to commit the transaction and the transaction becomes InDoubt.</source>
          <target state="translated">트랜잭션을 커밋하는 하려고 시도 하 고 트랜잭션이 indoubt이 예외가 throw 됩니다.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Transactions.TransactionInDoubtException">
          <source>This is a recoverable error.</source>
          <target state="translated">복구할 수 있는 오류입니다.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="T:System.Transactions.TransactionInDoubtException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that contains a message that explains why the exception occurred.</source>
          <target state="translated">예외가 발생한 이유를 설명하는 메시지가 포함된 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> class with the specified message.</source>
          <target state="translated">지정된 메시지를 사용하여 <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object that describes a failed serialization.</source>
          <target state="translated">실패한 serialization을 설명하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 개체입니다.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object that describes a failed serialization context.</source>
          <target state="translated">실패한 serialization 컨텍스트를 설명하는 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 개체입니다.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> class with the specified serialization and streaming context information.</source>
          <target state="translated">지정된 serialization과 스트리밍 컨텍스트 정보를 사용하여 <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that contains a message that explains why the exception occurred.</source>
          <target state="translated">예외가 발생한 이유를 설명하는 메시지가 포함된 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)">
          <source>Gets the exception instance that causes the current exception.</source>
          <target state="translated">현재 예외를 발생시키는 예외 인스턴스를 가져옵니다.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)">
          <source>For more information, see the <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">자세한 내용은 <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> 속성을 참조하세요.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> class with the specified message and inner exception.</source>
          <target state="translated">지정된 메시지 및 내부 예외를 사용하여 <ph id="ph1">&lt;see cref="T:System.Transactions.TransactionInDoubtException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>